concepts/modularization/README.md

```diff
@@ -0,0 +1,170 @@
+
+
+# Example local and central operators
+In this example 2 components are defined:
+- central operator for compass integration
+- local operator for eventing
+
+![](assets/operators-example.drawio.svg)
+
+# FAQ
+
+## Do we still release Kyma? What is a Kyma release?
+ 
+Kyma release contains `kyma-operator` in the given version and component descriptors for release channels. As we can continuously release components and upgrade their versions in the release channels the Kyma release is not so important anymore, but it can be useful for the open-source community.
+
+## Can I still use `kyma deploy` command to install Kyma in my cluster?
+
+Yes, but under the hood, `kyma-operator` will be used to install component operators. It is not decided yet if kyma CLI will install `kyma-operator` in the cluster or will contain `kyma-operator` code and run it locally.
+
+## I have a simple component with helm chart. Why do I need an operator?
+
+With the operator you can fully control your component lifecycle and ensure that your component will be reconciled  to the desired state (watch component config and actual state).
+
+## I don't know how to write the operator. Can I use some generic operator for installing my chart?
+
+Yes. You can use [Operator SDK - Helm](https://sdk.operatorframework.io/docs/building-operators/helm/) to generate it from your charts. You can create Helm based operator in a few minutes. If you want to better control the operator logic using [Operator SDK - Go](https://sdk.operatorframework.io/docs/building-operators/golang/) or [Kubebuilder](https://book.kubebuilder.io/)
+
+## Why should I provide a central operator
+
+Consider providing central operator when:
+- you deal with resources outside of Kyma cluster
+- you need access to external systems/resources with powerful credentials (that cannot be stored in the Kyma cluster)
+
+## How to roll out a new module version in phases?
+
+Use release channels to push the new version in the rapid channel first and after some time you can push that version to the stable channel. Release channels are flexible and if you need to test the new version only on 1 cluster you can create a new release channel and assign only one cluster to that channel. 
+
+## Can I run multiple versions of central operator
+
+Yes. But you have to ensure that each module instance described by your module custom resource is reconciled (managed) by a single operator to avoid concurrent updates and unpredictable outcomes. You can achieve that by marking your custom resources with labels pointing to release channels in your module template. Then you can deploy one central module per release channel and update them independently.
```

Comment:
```suggestion
Yes. But to avoid concurrent updates and unpredictable outcomes, you must ensure that each module instance described by your module custom resource is reconciled (managed) by a single operator. You can achieve that by marking your custom resources with labels pointing to release channels in your module template. Then you can deploy one central module per release channel and update them independently.
```